bin/projects/dbatools/dbatools/Commands/SelectDbaObjectCommand.cs

```diff
@@ -7,7 +7,7 @@
 namespace Sqlcollaborative.Dbatools.Commands
 {
     /// <summary>
-    /// Implements the Select-DbaObject command
+    /// Implements the <c>Select-DbaObject</c> command
```

Comment:
I hope that renders well in the documents, because that's the right way to do inline code in xml comments.